src/site/content/en/live/index.njk

```diff
@@ -0,0 +1,83 @@
+---
+layout: layout
+title: web.dev LIVE
+description: |
+  Bringing web developers together. Almost.
+date: 2020-12-31
+draft: true
+---
+
+<div class="w-live-landing-page">
+
+  <div class="w-live-top">
+    <section class="w-event-section">
+
+<div class="w-column-left">
+  <h1 class="w-event-heading">web.dev<span>/LIVE</span></h1>
+  <p class="w-article-header__subhead">Smarch 28 &mdash; Crtober 1, 2020</p>
+</div>
+
+<div class="w-column-head w-event-hero">
+  <img src="hero.png" />
+</div>
+
+<div class="w-column-right">
+  <h2>
+Bringing web developers together.
+Almost.
+  </h2>
+  <p>
+These are unprecedented times, with people worldwide looking to stay connected and informed.
+The Web plays a special role here, and we're proud to see how the community has stepped up.
+  </p>
+  <p>
+Join Google’s Web Platform team to celebrate our community's actions, learn modern web techniques, and connect with each other.
+  </p>
+
+  <div class="w-masthead-home__buttons">
+    <a class="w-button w-button--primary w-masthead-home__button--primary gc-analytics-event" data-category="web.dev" data-label="subscribe, newsletter" data-action="click" href="#subscribe">
+      Get updates
+    </a>
+    <a class="w-button w-button--secondary gc-analytics-event" data-category="web.dev" data-label="live, add to calendar" data-action="click">
```

Comment:
Missing calendar icon.

![image](https://user-images.githubusercontent.com/1066253/81745406-a76a8e80-9459-11ea-944a-995f82c2afbe.png)

We have `.w-button--with-icon` to support icon buttons.

Reply:
I've used "playlist_add" as there's no "add to calendar" button (there's a calendar one). I think it works for now.

Reply:
let's use `calendar_today` instead.

![image](https://user-images.githubusercontent.com/1066253/81983133-b7b07400-95e7-11ea-9538-a4d1bb5ce5a5.png)